tests/unit/plot/test_mode_detector.py

```
@@ -0,0 +1,340 @@
+        standalone.mkdir()
+        jsonl_file = standalone / "profile_export.jsonl"
+        with open(jsonl_file, "w") as f:
+            for record in sample_jsonl_data:
+                f.write(f"{record}\n")
+
+        # Should find 3 runs total: outer, inner, standalone
+        runs = mode_detector.find_run_directories([nested_run_dirs])
+        assert len(runs) == 3
+
+    def test_deeply_nested_runs(
+        self, mode_detector: ModeDetector, tmp_path: Path, sample_jsonl_data
```

Ruff (0.14.4)

Unused method argument: `sample_jsonl_data`

(ARG002)